src/browse.js

```diff
@@ -47,9 +47,9 @@ var browse = {
 
     constructURL:(keyword, users) => {
```

Comment:
will this work if input contains multiple keywords? a scenario such as user does a search for "ONNX image recognization", in that case, do you need to have multiple "topic:" parameters, or do you need to escape the search phase like ONNX%20image%20recognization?

Reply:
If the user inputs "ONNX image recognization", the current implementation will just pass that string as a single keyword and search for matches to this string